### PR TITLE
fix: sort region nodes by node name

### PR DESCRIPTION
**Pull request by chasers**
Sorts region nodes by node name in case the node list order is unstable.

### CODE REVIEW

File: lib/realtime/postgres_cdc.ex [hunk 1]
```diff
@@ -78,9 +78,15 @@ defmodule Realtime.PostgresCdc do
...
-  @spec region_nodes(String.t()) :: [{pid, any}]
+  @doc """
+  Lists the nodes in a region. Sorts by node name in case the list order
+  is unstable.
+  """
+
+  @spec region_nodes(String.t()) :: [{pid(), [node: atom()]}]
   def region_nodes(region) when is_binary(region) do
     :syn.members(RegionNodes, region)
+    |> Enum.sort_by(fn {_pid, [node: node]} -> node end)
```
Comment (abc3) [inline lib/realtime/postgres_cdc.ex:89]: Should we update the current function to return only a list of nodes? Pid inside seems useless

Comment (chasers) [reply lib/realtime/postgres_cdc.ex:89]: Sure

**Review by abc3 — APPROVED** [review]
>should just error or start retrying until we have two launch nodes in a region to make this behavior more deterministic and/or until we support more replication slots

I don't think it's necessary, it's a pretty rare case

**Pull request merged**